### ISSUE TITLE
org.eclipse.jgit.api.errors.JGitInternalException: Missing unknown 0000000000000000000000000000000000000000

### ISSUE THREAD

**Issue by mslinn** [sbt/sbt-fresh#9]
First time I run:

````
$ sbt fresh
[info] Loading global plugins from /home/mslinn/.sbt/0.13/plugins
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[info] Loading project definition from /var/work/training/project
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
org.eclipse.jgit.api.errors.JGitInternalException: Missing unknown 0000000000000000000000000000000000000000
	at org.eclipse.jgit.api.StatusCommand.call(StatusCommand.java:154)
	at com.typesafe.sbt.git.JGit.hasUncommittedChanges(JGit.scala:84)
	at com.typesafe.sbt.SbtGit$$anonfun$buildSettings$8$$anonfun$apply$8.apply(SbtGit.scala:116)
	at com.typesafe.sbt.SbtGit$$anonfun$buildSettings$8$$anonfun$apply$8.apply(SbtGit.scala:116)
	at com.typesafe.sbt.git.DefaultReadableGit.withGit(ReadableGit.scala:35)
	at com.typesafe.sbt.SbtGit$$anonfun$buildSettings$8.apply(SbtGit.scala:116)
	at com.typesafe.sbt.SbtGit$$anonfun$buildSettings$8.apply(SbtGit.scala:116)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:175)
	at sbt.EvaluateSettings$INode.evaluate(INode.scala:135)
	at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$submitEvaluate$1.apply$mcV$sp(INode.scala:69)
	at sbt.EvaluateSettings.sbt$EvaluateSettings$$run0(INode.scala:78)
	at sbt.EvaluateSettings$$anon$3.run(INode.scala:74)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.jgit.errors.MissingObjectException: Missing unknown 0000000000000000000000000000000000000000
	at org.eclipse.jgit.internal.storage.file.WindowCursor.open(WindowCursor.java:145)
	at org.eclipse.jgit.lib.ObjectReader.open(ObjectReader.java:226)
	at org.eclipse.jgit.revwalk.RevWalk.parseAny(RevWalk.java:859)
	at org.eclipse.jgit.revwalk.RevWalk.parseTree(RevWalk.java:799)
	at org.eclipse.jgit.lib.IndexDiff.<init>(IndexDiff.java:315)
	at org.eclipse.jgit.lib.IndexDiff.diff(IndexDiff.java:554)
	at org.eclipse.jgit.lib.IndexDiff.diff(IndexDiff.java:377)
	at org.eclipse.jgit.api.StatusCommand.call(StatusCommand.java:148)
	at com.typesafe.sbt.git.JGit.hasUncommittedChanges(JGit.scala:84)
	at com.typesafe.sbt.SbtGit$$anonfun$buildSettings$8$$anonfun$apply$8.apply(SbtGit.scala:116)
	at com.typesafe.sbt.SbtGit$$anonfun$buildSettings$8$$anonfun$apply$8.apply(SbtGit.scala:116)
	at com.typesafe.sbt.git.DefaultReadableGit.withGit(ReadableGit.scala:35)
	at com.typesafe.sbt.SbtGit$$anonfun$buildSettings$8.apply(SbtGit.scala:116)
	at com.typesafe.sbt.SbtGit$$anonfun$buildSettings$8.apply(SbtGit.scala:116)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.EvaluateSettings$MixedNode.evaluate0(INode.scala:175)
	at sbt.EvaluateSettings$INode.evaluate(INode.scala:135)
	at sbt.EvaluateSettings$$anonfun$sbt$EvaluateSettings$$submitEvaluate$1.apply$mcV$sp(INode.scala:69)
	at sbt.EvaluateSettings.sbt$EvaluateSettings$$run0(INode.scala:78)
	at sbt.EvaluateSettings$$anon$3.run(INode.scala:74)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[error] org.eclipse.jgit.api.errors.JGitInternalException: Missing unknown 0000000000000000000000000000000000000000
````

**Comment by mslinn** [comment]
Cannot duplicate. This error was caused by another problem, unrelated

**Comment by talgendler** [comment]
The exact same thing happened to me as well, I'm running on OSX.
Have "git" in PATH by the way. Perhaps it's related to [this](https://www.eclipse.org/forums/index.php/t/847612/)

**Comment by talgendler** [comment]
Problem fixed after updating SBT from 0.13.9 to 0.13.11
